.github/actions/prepare-backend/action.yml

```diff
@@ -28,7 +28,7 @@ runs:
     - name: Fetch CI config
       shell: bash
       run: |
-        curl -o ci-test-config.json https://raw.githubusercontent.com/metabase/metabase/refs/heads/master/ci-test-config.json
+        curl -o ci-test-config.json https://raw.githubusercontent.com/metabase/ci-test-config/refs/heads/master/ci-test-config.json
```

Comment:
Let's get rid of this whole thing from `prepare-backend` if it's only ever going to be used in test-driver action.

Reply:
Good idea